### ISSUE TITLE
JAWS (18.0.5038) loses "forms mode" for legacy mat-form-field with floatLabel="never"

### ISSUE THREAD

**Issue by ampled** [angular/components#14927]
#### What is the expected behavior?
JAWS should be able to retain "Forms mode" (ignore jaws shortcuts) when typing in an input-field.

#### What is the current behavior?
After typing a letter (causing the label to go invisible) in a text input field JAWS is thrown out of "forms mode" (indicated by a pop sound). This happens for mat-form-fields with legacy appearance with the floatLabel-input set to "never"

#### What are the steps to reproduce?
- use JAWS screen reader
- Navigate to form field with tab or keyboard arrows (the form field should have the input floatLabel set to "never")
- Listen for the "pop" sound that indicates JAWS is now in forms mode (when forms mode is set to AUTO)
- type a letter
- be sad when JAWS shortcuts get triggered when you try to type 

https://jaws-forms-mode-mat-form-field.stackblitz.io/

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
reproduced in
Angular: 7.0.1 and 7.2.0
Material: 7.0.2 and 7.2.0
Typescript: 3.2.2
Chrome 71 and JAWS (18.0.5038)

**Comment by mmalerba** [comment]
The legacy appearance is not intended to be used by new code. It was left in to avoid breaking old code and will eventually be removed. Therefore closing this issue
